### PR TITLE
New endpoints for Jira

### PR DESCRIPTION
**Pull request by dzschille**
PR is based on #13. It fixes the authorization check for Jira and restores the old endpoints for Confluence.

For more details about new endpoints, see https://jira.atlassian.com/browse/JRACLOUD-67169

### CODE REVIEW

File: backup.sh [hunk 1]
```diff
@@ -74,23 +81,50 @@ if [ $? -ne 0 ]; then
...
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json, text/javascript, */*; q=0.01' \
+    -H 'X-Requested-With: XMLHttpRequest' \
+    --data-binary "{\"cbAttachments\":\"${ATTACHMENTS}\", \"exportToCloud\":\"true\" }" )
 
 # Checks if we were authorized to create a new backup
-if [ "$(echo "$BKPMSG" | grep -c Unauthorized)" -ne 0 ]  || [ "$(echo "$BKPMSG" | grep -ic "<status-code>401</status-code>")" -ne 0 ]; then
-    echo "ERROR: authorization failure"
-    exit
+if [ $FILEPREFIX = "JIRA" ]; then
```
Comment (jodrescher) [inline backup.sh:93]: In this section there should be something like this, to abort when to many tries:
```
if [ "$(echo "$BKPMSG" | grep -ic "Backup frequency is limited")" -ne 0 ]; then
      echo "ERROR: Backup frequency is limited"
      exit
fi
```

Comment (jodrescher) [reply backup.sh:93]: certainly the download should work although, so "exit" is not required, but it is good to know

Comment (dzschille) [reply backup.sh:93]: @jodrescher , good idea, to handle the case when you reach a limit. Feel free to add this feature into another PR.

**Comment by mattock** [comment]
I finally got around to testing the latest code, including this PR. Both JIRA and Confluence backups seem to work great, thanks to all!

**Pull request merged**